### ISSUE TITLE
Not able to get json from finger

### ISSUE THREAD

**Issue by krionical** [pfefferle/wordpress-activitypub#126]
This is my finger request: [https://webfinger.net/lookup/?resource=joebs%40jbmda.com](https://webfinger.net/lookup/?resource=joebs%40jbmda.com)

My blog is in a subdirectory folder called jbmdacom which is a subdirectory of jbmdacom in the public.html folder. So I have this line in .htaccess--

RedirectMatch "^\/\.well-known(.*)$" "\/jbmdacom/jbmdacom\/\.well-known$1" in the public.html folder, and tried 
RedirectMatch "^\/\.well-known(.*)$" "\/jbmdacom/\.well-known$1" relative to the first jbmdacom.

I can't seem to get any success so I'm not sure what I am doing wrong and would appreciate any insight you could offer.

**Comment by pfefferle** [comment]
I think you have use the full URL for your redirect target.

If you try to access the url in the browser: `https://jbmda.com/.well-known/webfinger?resource=acct%3Ajoebs%40jbmda.com` it redirects to: `https://jbmdacom/jbmdacom///.well-known/webfinger?resource=acct%3Ajoebs%40jbmda.com`

**Comment by pfefferle** [comment]
The second problem is, that it does not add the correct "nice permalinks": https://jbmda.com/jbmdacom/.well-known/webfinger?resource=acct%3Ajoebs%40jbmda.com

So either you have to configure them by hand or you have to redirect to: the wp-json API endpoint, that also does not really seem to work on your site: `https://jbmda.com/jbmdacom/wp-json/activitypub/1.0/webfinger?resource=acct%3Ajoebs%40jbmda.com`

**Comment by pfefferle** [comment]
Next problem... Your profile URLs are also not accessible, but needed for AP: `https://jbmda.com/jbmdacom/author/joebs/`

**Comment by krionical** [comment]
I am using your theme which I am sure you probably realize. Is this a theme issue or a host issue? The theme is Autonomie and I just performed a basic wordpress install and then uploaded the theme. I tried the link suggestions above but nothing appears to be working. It sounds like this might be "too much trouble" for whatever reason and I do not want you to spend your time frivolously if I am the only one having this issue. I'll just disable the plugin and not be social.

**Comment by pfefferle** [comment]
> I am using your theme which I am sure you probably realize

I did :)

> Is this a theme issue or a host issue?

It is not a theme, but a problem with WordPress and/or your hoster. You can try to hit the save-button on the permalinks page again, then WordPress will regenerate them. Maybe this helps.

BTW: The URL that works is:

```
https://jbmda.com/jbmdacom/?rest_route=/activitypub/1.0/webfinger&resource=acct%3Ajoebs%40jbmda.com
```

**Comment by krionical** [comment]
Well, thanks for your help then. If it is the host short of reinstalling the thing I am not sure what would help--and I don't intend to reinstall all of it since I have all ready put a lot of effort into customizing this one.

**Comment by krionical** [comment]
An update: It was partly the theme. I was able to use your plugin with another theme, and got it to working after a re-install of wordpress and everything else. The theme, for some reason, did not install smoothly and duplicated links and all other manner of oddities. I kinda took it as a sign to do something else, so I did. Might just be on my host, might not be, but figured I'd give you a heads up.
